component/client-handler/src/main/java/org/wso2/carbon/identity/oauth2/token/handler/clientauth/jwt/PrivateKeyJWTClientAuthenticator.java

```diff
@@ -106,6 +110,15 @@ private void readServerConfig(){
     public boolean authenticateClient(HttpServletRequest httpServletRequest, Map<String, List> bodyParameters,
                                       OAuthClientAuthnContext oAuthClientAuthnContext) throws OAuthClientAuthnException {
 
+        //   For FAPI compliant applications the allowed JWT signing algorithm should be registered at the application
```

Comment:
Add multi-line comments

Reply:
Fixed in 2c2c165facbf4157db50b88f2765fbc9cee49fe0